pydeconz/group.py

```diff
@@ -136,6 +136,19 @@ def async_add_scenes(self, scenes, async_set_state_callback):
             if scene['id'] not in self._scenes
         }
 
+    def sync_color_state(self, light):
+        """Sync color state with light."""
+        x, y = light.xy
+        self.async_update({'state': dict(
```

Comment:
For consistencies sake; I think it'd be better to use {}, what do you think?

Reply:
I'm not a fan of big dicts with {} as I think the dict() syntax looks cleaner, but I can change this if you want it for consistency. Or I could change both to dict(), it's your decision.

Reply:
It makes me uneasy, change it to {}.